### PR TITLE
Enable batch upload of labels / descriptions

### PR DESCRIPTION
**Pull request by Vesihiisi**
Use 0.3.3 of wdstuff to batch upload labels and descriptions.

Task: https://phabricator.wikimedia.org/T167280

### CODE REVIEW

File: importer/Uploader.py [hunk 1]
```diff
@@ -11,86 +11,34 @@ class Uploader(object):
...
-            self.wdstuff.add_description(
-                lang, desc_content, target_item)
-            if log:
-                t_id = target_item.getID()
-                message = "{} ADDED DESCRIPTION {} {}".format(
-                    t_id, lang, desc_content)
-                log.logit(message)
+        """Add descriptions."""
+        descriptions_for_upload = {}
+        for language_key in descriptions:
```
Comment (lokal-profil) [inline importer/Uploader.py]: couldn't you loop over `for language_key, desc_content in descriptions.items():`?

Comment (lokal-profil) [inline importer/Uploader.py]: Actually how is `descriptions` different from `descriptions_for_upload` at the end of the loop?

Comment (lokal-profil) [inline importer/Uploader.py]: Both of these also apply for `add_labels` of course

Comment (Vesihiisi) [inline importer/Uploader.py]: Hm, I think that's a leftover from the previous way of uploading them, and how make_labels() worked.

Comment (Vesihiisi) [inline importer/Uploader.py]: Since the data from Monument was first processed through make_labels() before it reached add_labels()

Comment (Vesihiisi) [inline importer/Uploader.py]: Fixed by simplifying this further....

File: importer/Uploader.py [hunk 2]
```diff
@@ -13,33 +13,21 @@ class Uploader(object):
...
-            label_content = labels[language_key]
-            labels_for_upload[language_key] = label_content
-        self.wdstuff.add_multiple_label_or_alias(
-            labels_for_upload, target_item)
+        self.wdstuff.add_multiple_label_or_alias(labels, target_item)
         if log:
             t_id = target_item.getID()
             message = "{} ADDED LABELS {}".format(
-                t_id, labels_for_upload)
+                t_id, labels)
```
Comment (lokal-profil) [inline importer/Uploader.py]: Probably no need for a line-break here anymore

**Pull request merged**